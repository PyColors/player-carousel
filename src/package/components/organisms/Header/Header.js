import React from "react";

import Container from "muicss/lib/react/container";
import Row from 'muicss/lib/react/row';
import Col from 'muicss/lib/react/col'

import logo from '../../../../../public/bt_logo.png';
import lock from '../../../../../public/lock.png';

const Header = () => (
    <header className='header'>
        <Container fluid={true}>
            <Row>
                <Col xs="6">
                    <img className='header__logo' src={logo} alt='BT' />
                    <h1 className='header__title'>BT TV</h1>
                </Col>
                <Col xs="6 mui--text-right">
                    <img className='header__lock' src={lock} alt='BT' />
                    <a href='#' className='header__link-lock'>Log in</a>
                </Col>
            </Row>
        </Container>

    </header>
);
export default Header;
