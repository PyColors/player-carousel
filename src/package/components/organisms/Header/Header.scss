.inline-middle {
  display: inline-block;
  vertical-align: middle;
}

.header {
  margin: 20px 0 20px;
  font-family: "BT Regular", serif;

  &__logo {
    width: 8%;
    @extend .inline-middle;
  }

  &__title {
    @extend .inline-middle;
    margin: 0 0 0 10px;
    color: #fff;
    font-size: 16px;
  }

  &__lock {;
    @extend .inline-middle;
    width: 3%;
  }

  &__link-lock {
    @extend .inline-middle;
    margin: 0 0 0 10px;
    color: #5D5A62;
  }

}
