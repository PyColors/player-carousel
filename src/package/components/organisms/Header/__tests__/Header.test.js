import React from "react";
import Header from "../Header";

it("Should renders Header page without exploding", () => {
    const wrapper = shallow(<Header>Hello Jest!</Header>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveLength(1)
});
