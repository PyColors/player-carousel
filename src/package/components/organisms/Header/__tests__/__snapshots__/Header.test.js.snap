// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders Header page without exploding 1`] = `
<header
  className="header"
>
  <Container
    className=""
    fluid={true}
  >
    <Row
      className=""
    >
      <Col
        className=""
        lg={null}
        lg-offset={null}
        md={null}
        md-offset={null}
        sm={null}
        sm-offset={null}
        xl={null}
        xl-offset={null}
        xs="6"
        xs-offset={null}
      >
        <img
          alt="BT"
          className="header__logo"
          src="test-file-stub"
        />
        <h1
          className="header__title"
        >
          BT TV
        </h1>
      </Col>
      <Col
        className=""
        lg={null}
        lg-offset={null}
        md={null}
        md-offset={null}
        sm={null}
        sm-offset={null}
        xl={null}
        xl-offset={null}
        xs="6 mui--text-right"
        xs-offset={null}
      >
        <img
          alt="BT"
          className="header__lock"
          src="test-file-stub"
        />
        <a
          className="header__link-lock"
          href="#"
        >
          Log in
        </a>
      </Col>
    </Row>
  </Container>
</header>
`;
